engine/modules/opentelemetry/src/otl_config.cc

```diff
@@ -80,8 +81,14 @@ static constexpr std::string_view _grpc_config_schema(R"(
 otl_config::otl_config(const std::string_view& file_path,
                        asio::io_context& io_context) {
   static json_validator validator(_grpc_config_schema);
-  rapidjson::Document file_content_d =
-      rapidjson_helper::read_from_file(file_path);
+  rapidjson::Document file_content_d;
+  try {
+    file_content_d = std::move(rapidjson_helper::read_from_file(file_path));
```

Comment:
clang will raise a warning telling that the std::move is not needed, because of C++ optimizations

Reply:
done